core-bundle/src/Resources/views/Frontend/preview_toolbar_base_js.html.twig

```diff
@@ -177,6 +177,24 @@
     .cto-toolbar .tl_submit:active {
         color: #aaa;
     }
+
+    .cto-toolbar .badge-title {
+        align-self: flex-start;
```

Comment:
Top left on mobile seems good to me, yes I did mean on desktop centered in the bar. Not sure if that CSS does that 😂  if you maybe have another screenshot at hand?

Reply:
With the proposed change it would look like this:
![image](https://user-images.githubusercontent.com/14016098/132988156-3d1423e7-7ab8-4d1a-b543-9836a917738e.png)

Reply:
looks good to me